brkt_cli/make_user_data/__init__.py

```
@@ -25,16 +25,27 @@
 )
 log = logging.getLogger(__name__)
 
+DEFAULT_CLOUDINIT_FORMAT = 'text/x-shellscript'
 
-def _add_files_to_instance_config(instance_cfg, files_list):
+
+def _add_files_to_instance_config(instance_cfg, files_list,
+                                  guest_files=False):
     for fname in files_list:
+        if guest_files:
+            if ':' in fname:
```

I don't understand.  What's the downside of making the code more robust?  The behavior isn't different if you use a regexp.  Without the regexp, if someone has a colon in their filename, it will lead to silent failure.